### ISSUE TITLE
Broken on 1.3

### ISSUE THREAD

**Issue by pitaj** [psychobunny/nodebb-plugin-custom-pages#41]
@psychobunny 

https://community.nodebb.org/topic/1701/nodebb-plugin-custom-pages-custom-static-pages-for-nodebb/86

**Comment by Xseba360** [comment]
Still broken on 1.4 as well.
```
Internal Error.
Oops! Looks like something went wrong!
/forums/admin/custom-pages
Failed to lookup view "admin/custom-pages" in views directory "/usr/src/app/public/templates"
```

**Comment by psychobunny** [comment]
thanks, fixed for 1.5.3 from https://github.com/psychobunny/nodebb-plugin-custom-pages/pull/47
